teams/rust-analyzer.toml

```diff
@@ -7,7 +7,6 @@ members = [
     "lnicola",
     "Veykril",
     "flodiebold",
-    "bjorn3",
```

Comment:
Should this be removed to `alumni` or was the inclusion a mistake?

Reply:
That was intentional, `bjorn3` was only in the team for issue triaging permissions when we removed the contributors team. I've moved them to the contributors back now with this.

Reply:
LGTM